includes/sslprotocols-none-md.md

```diff
@@ -0,0 +1 @@
+﻿As of .NET 4.7, this method authenticates using <xref:System.Security.Authentication.SslProtocols.None>, which allows the operating system to choose the best protocol to use, and to block protocols that are not secure. In .NET 4.6 (and .NET 4.5 with the latest security patches installed), the allowed TLS/SSL protocols versions are 1.2, 1.1, and 1.0 (unless you disable strong cryptography by editing the Windows Registry).
```

Comment:
.NET -> .NET Framework

I'd also future-proof the sentence by saying: Starting with .NET Framework 4.7